test/e2e/specs/specs-playwright/wp-support__home-spec.ts

```diff
@@ -0,0 +1,77 @@
+import { DataHelper, LoginFlow, SupportComponent, setupHooks } from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( 'Support: My Home' ), function () {
+	let page: Page;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	describe.each( [
+		{ siteType: 'Simple', user: 'defaultUser' },
+		{ siteType: 'Atomic', user: 'wooCommerceUser' },
+	] )( 'Search from Support Card ($siteType)', function ( { user } ) {
+		let supportComponent: SupportComponent;
+
+		it( 'Log in', async function () {
+			const loginFlow = new LoginFlow( page, user );
+			await loginFlow.logIn();
+		} );
+
+		it( 'Displays default entries on card', async function () {
+			supportComponent = new SupportComponent( page );
+			await supportComponent.showSupportCard();
+			const defaultResults = await supportComponent.getOverallResultsCount();
+			expect( defaultResults ).toBeGreaterThanOrEqual( 5 );
+		} );
+
+		it( 'Enter valid search keyword', async function () {
+			const keyword = 'Domain';
+			await supportComponent.search( keyword );
+		} );
+
+		it( 'Search results are shown', async function () {
+			const supportCount = await supportComponent.getSupportResultsCount();
+			expect( supportCount ).toBe( 4 );
+
+			const adminCount = await supportComponent.getAdminResultsCount();
+			expect( adminCount ).toBe( 3 );
```

Comment:
Do you think using specific numbers will add fragility over time, in case new articles are added? Would it be better to just make sure that _some_ support and admin results are returned?

Reply:
If I recall correctly, the number of results rendered on page _does_ vary depending on the state of the account. I do not have the exact thread where this discussion took place, but it was mentioned that presence of a pending or active plan on the account impacts result count of some search queries (eg. `domain`). The checks for those results are `greaterThanOrEqual to 6` for this reason.

Now, as for the `admin results` and `support article results`, I am less inclined to check for a `>= or <=` because if the number of results change, it can only result from either of the following:
- explicit change in the branch, or;
- test params have been changed (test account, plans provisioned on the account, etc).

Seeing the second is not likely to occur for this test (this test does not modify user state, so no reason to use site isolation) the plan is to continue using `defaultUser` for the foreseeable future. Unless the active plan on the account is cancelled/lapsed, the plan component does not change either.

That leaves explicit change in the feature branch as the only reason why this check would fail.

My bad for the wall of text 😓

Reply:
No worries, it was well-written! 😄  That sounds good to me!